build/targets.mk

```diff
@@ -319,7 +319,7 @@ endif
 ifeq ($(TARGET),ANDROID)
   ANDROID_NDK ?= $(HOME)/opt/android-ndk-r25c
 
-  ANDROID_SDK_PLATFORM = android-30
+  ANDROID_SDK_PLATFORM = android-33
```

Comment:
Do we need a tangible reason?
I had the impression that we update libraries / compilers from time to time, even when the old ones still work perfectly fine.
If that's not the case this pr is completely pointless, as there is no immediate advantage of updating.

Should i remove the commit with the android-33 platform and the rest is fine?

Reply:
Libraries are updated to get bug (vulnerability) fixes (and rarely for new features).
But updating the Android SDK platform is different; it doesn't fix any bugs, because that's just the API declaration; the implementation is not part of that.
So yes, this is a serious question: why should be raise the API version?

Reply:
See a55121e3f415983dfe54f10de5ca098ec5c45924 for what could be a good reason to raise the version, and for what problems could arise.
In any case, it makes no sense to raise the SDK platform but not the targetSdkVersion.

Reply:
ok, i see, things like that really don't help to like Android...

Reply:
Android is a terrible shitshow, due to Google politics. It used to be not that way, but then came market dominance.